app/mci/cmd/create_test.go

```diff
@@ -109,18 +118,19 @@ func TestCreateIngress(t *testing.T) {
 			Output: "context-1\ncontext-2",
 			Err:    nil,
 		},
-		{
-			Args:   []string{"kubectl", "--kubeconfig=kubeconfig", "create", "--filename=../../../testdata/ingress.yaml", "--context=context-1"},
-			Output: "ingress created",
-			Err:    nil,
-		},
-		{
-			Args:   []string{"kubectl", "--kubeconfig=kubeconfig", "create", "--filename=../../../testdata/ingress.yaml", "--context=context-2"},
-			Output: "ingress created",
-			Err:    nil,
-		},
 	}
-	if err := run(expectedCommands, runFn); err != nil {
+	if err := run(&fakeClient, expectedCommands, runFn); err != nil {
 		t.Errorf("%s", err)
 	}
+	actions := fakeClient.Actions()
+	if len(actions) != 2 {
+		t.Errorf("Expected 2 actions: Create Ingress 1, Create Ingress 2. Got:%v", actions)
+	}
+	if !actions[0].Matches("create", "ingresses") {
+		t.Errorf("Expected ingress creation.")
+	}
+	// TODO(G-Harmon): Do we need to verify anything in the Ingresses?
```

Comment:
optional: You can run action.GetObject to verify that it matches the expected ingress.